sql/updates/auth/2015_MM_DD_NN_auth.sql

```diff
@@ -0,0 +1,532 @@
+(66847),
+(66906),
+(67336),
+(67466),
+(68056),
+(68057),
+(68187),
+(68188),
+(69395),
+(69820),
+(69826),
+(71342),
+(71810),
+(72286),
+(72807),
+(72808),
+(73313),
+(73629),
+(73630),
+(74856),
+(74918),
+(75207),
+(75596),
+(75614),
+(75973),
+(84751),
+(87090),
+(87091),
+(88331),
+(88335),
+(88718),
+(88741),
+(88742),
+(88744),
+(88746),
+(88748),
+(88749),
+(88750),
+(88990),
+(90621),
+(92155),
+(92231),
+(92232),
+(93326),
+(93623),
+(93644),
+(96491),
+(96499),
+(96503),
+(97359),
+(97493),
+(97501),
+(97560),
+(97581),
+(98204),
+(98718),
+(98727),
+(100332),
+(100333),
+(101282),
+(101542),
+(101573),
+(101821),
+(102346),
+(102349),
+(102350),
+(102488),
+(102514),
+(103081),
+(103195),
+(103196),
+(107203),
+(107516),
+(107517),
+(107842),
+(107844),
+(107845),
+(110039),
+(110051),
+(113120),
+(113199),
+(118089),
+(118737),
+(120043),
+(120395),
+(120822),
+(171847),
+(121820),
+(121836),
+(121837),
+(121838),
+(121839),
+(122708),
+(123182),
+(123886),
+(123992),
+(123993),
+(124408),
+(124550),
+(124659),
+(126507),
+(126508),
+(127154),
+(127156),
+(127158),
+(127161),
+(127164),
+(127165),
+(127169),
+(127170),
+(127174),
+(127176),
+(127177),
+(127209),
+(127213),
+(127216),
+(127220),
+(127271),
+(127286),
+(127287),
+(127288),
+(127289),
+(127290),
+(127293),
+(127295),
+(127302),
+(127308),
+(127310),
+(129552),
+(129918),
+(129932),
+(129934),
+(129935),
+(130086),
+(130092),
+(130137),
+(130138),
+(130965),
+(130985),
+(132036),
+(132117),
+(132118),
+(132119),
+(133023),
+(134359),
+(134573),
+(135416),
+(135418),
+(136163),
+(136164),
+(136400),
+(136471),
+(136505),
+(138423),
+(138424),
+(138425),
+(138426),
+(138640),
+(138641),
+(138642),
+(138643),
+(139407),
+(139442),
+(139448),
+(139595),
+(140249),
+(140250),
+(142073),
+(142266),
+(142478),
+(142641),
+(142878),
+(146615),
+(146622),
+(148392),
+(148396),
+(148417),
+(148428),
+(148476),
+(148618),
+(148619),
+(148620),
+(148970),
+(148972),
+(149801),
+(153489),
+(163024),
+(163025),
+(155741),
+(169952),
+(170347),
+(171436),
+(171617),
+(171620),
+(171621),
+(171622),
+(171623),
+(171624),
+(171625),
+(171626),
+(171627),
+(171629),
+(171632),
+(171634),
+(171635),
+(171636),
+(171637),
+(171638),
+(171824),
+(171825),
+(171828),
+(171829),
+(171830),
+(171831),
+(171832),
+(171833),
+(171834),
+(171835),
+(171836),
+(171838),
+(171839),
+(171841),
+(171842),
+(171843),
+(171844),
+(171845),
+(171846),
+(171848),
+(171849),
+(171851),
+(175700),
+(179244),
+(179245),
+(179478),
+(180545);
+
+INSERT INTO `account_mounts` (`accountId`, `spell`, `favorite`)
+SELECT DISTINCT `chars`.`account`, `characters`.`character_spell`.`spell`, 0 FROM `characters`.`character_spell`
+JOIN `characters`.`characters` `chars` ON `chars`.`guid` = `characters`.`character_spell`.`guid`
+WHERE `characters`.`character_spell`.`spell` IN (SELECT `spell` FROM `temp_mounts`);
+
+DELETE FROM `characters`.`character_spell` WHERE `spell` IN (SELECT `spell` FROM `temp_mounts`);
```

Comment:
@Aokromes I was writing about this. The reason why I don't want it in a seperate sql file is that it deletes the now useless mounts from character_spell and uses `temp_mounts` table which is dropped afterwards... -> it wouldn't do anything because the table temp_mounts wouldn't exist.
Yes I could just do "WHERE `spell` IN (1, 2,...)" but that would open up the possiblity that someone somewhere could execute the "DELETE FROM..." query before this one which would make him loose all playermounts.